pkg/codegen/testing/test/program_driver.go

```diff
@@ -97,6 +97,12 @@ var PulumiPulumiProgramTests = []ProgramTest{
 		Description: "Python regression test for #10914",
 		Skip:        allProgLanguages.Except("python"),
 	},
+	{
+		Directory:   "simplified-invokes",
+		Description: "Simplified invokes",
+		Skip:        codegen.NewStringSet("python", "go"),
+		SkipCompile: codegen.NewStringSet("dotnet", "nodejs"),
```

Comment:
I'm assuming that compiling is blocked on https://github.com/pulumi/pulumi/pull/11564. Is that right?

Reply:
Yeah waiting for that to be merged then gonna rebase this on top ✅

Reply:
I stand corrected. This is actually skipping program-gen compilation because `Pulumi.Std` and `@pulumi/std` are not real packages (yet). The sdk-gen already compiles both dotnet and nodejs, that's why it was compiling them (because it was only skipping compiling programs)

That said, I think it would be great to be able to test with the generated SDK by a project reference since we have a schema to go from but I don't think that's possible right now 🤔